asciidoctor-stylesheet/sass/components/_asciidoc.scss

```diff
@@ -352,6 +352,9 @@ b.button:after {
   //margin-bottom: $panel-margin-bottom * 2;
   border-bottom: $sect-divider-size $sect-divider-style $sect-divider-color;
   padding-bottom: 0.5em;
+  overflow: -moz-scrollbars-none;
+  position: sticky !important; 
```

Comment:
`position: sticky` is not making any difference here. Please apply it to `#header`. Also, `position: sticky` won't work on its own, you have to specify `top: 0` so the browser knows where to "stick" it when you scroll.